starknet_py/tests/e2e/docs/guide/test_account_sign_outside_transaction.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from starknet_py.net.client_models import TransactionFinalityStatus
+
+
+@pytest.mark.asyncio
+async def test_account_outside_execution_any_caller(
+    account,
+    argent_account,
+    map_contract,
+):
+    # pylint: disable=import-outside-toplevel,too-many-locals
+
+    # docs: start
+    import datetime
+
+    from starknet_py.constants import ANY_CALLER
+    from starknet_py.hash.selector import get_selector_from_name
+    from starknet_py.net.client_models import Call, OutsideExecutionTimeBounds
+
+    # Create a call to put value 20 under key 20. That will be executed
+    # as part of outside execution.
+
+    put_call = Call(
+        to_addr=map_contract.address,
+        selector=get_selector_from_name("put"),
+        calldata=[20, 20],
+    )
```

Comment:
nit: Maybe 1 -> 100? Seems like different key and value is a bit better for example purposes 😅 .
```suggestion
    # Create a call to put value 1 under key 100. That will be executed
    # as part of outside execution.

    put_call = Call(
        to_addr=map_contract.address,
        selector=get_selector_from_name("put"),
        calldata=[1, 100],
    )
```